src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,164 @@
+1. Overlaying element will get a new stacking context
+1. Filters may be combined
+
+{% Aside 'warning' %}
+  The backdrop-filter feature is computationally intensive, as are filters in general, so it may impact the performance of sites using it.
+{% endAside %}
+
+
+## Using backdrop-filter
+Get started in one line of CSS.
+
+### Single Filter
+
+```css
+.blur-behind-me {
+  backdrop-filter: blur(.5rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-rgb.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/netsi1964/pen/JqBLPK" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+### Combining Filters
+
+```css
+.brighten-saturate-and-blur-behind-me {
+  backdrop-filter: brightness(150%) saturate(150%) blur(1rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-ambient_blur.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/pepf/pen/GqZkdj" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+### iOS Frosted Notifications
+
+```css
+.frost {
+  backdrop-filter: blur(.25rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-ios_notifications.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://s.codepen.io/robinrendle/pen/ZoXGgG" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+
+## Conclusion
+Better late than never! Safari's had this feature since 2015. Regardless, Chrome's excited to release it in 76. The [Chromium bug](https://crbug.com/497522) has **560+ stars** on it, clearly marking it as a desired web design capability.
+
+Here's a final CSS API list and additional resources in case you're ready to start developing.
+
+### Overview
+```css
+/* Keyword value */
+backdrop-filter: none;
+
+/* URL to SVG filter */
+backdrop-filter: url(commonfilters.svg#filter);
+
+/* <filter-function> values */
+backdrop-filter: blur(2px);
+backdrop-filter: brightness(60%);
+backdrop-filter: contrast(40%);
+backdrop-filter: drop-shadow(4px 4px 10px blue);
+backdrop-filter: grayscale(30%);
+backdrop-filter: hue-rotate(120deg);
+backdrop-filter: invert(70%);
+backdrop-filter: opacity(20%);
+backdrop-filter: sepia(90%);
+backdrop-filter: saturate(80%);
+
+/* Multiple filters */
+backdrop-filter: url(filters.svg#filter) blur(4px) saturate(150%);
+
+/* Global values */
+backdrop-filter: inherit;
+backdrop-filter: initial;
+backdrop-filter: unset;
+```
+
+### One more demo
+A glimpse into the now possible in Chrome:
```

Comment:
```suggestion
A glimpse of what's now possible using `backdrop-filter`:
```

I'm not sure if this demo is strong enough to warrant its inclusion. The prior demos seem to cover it.

Reply:
nice to see them all together is what i felt. agree it could be pulled. will chat with other's about it 👍

Reply:
I completely agree with Rob.

Reply:
cool, it's gone then!